packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -2794,6 +2794,14 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
   // [assembleSemanticsNode] invocations.
   Queue<SemanticsNode>? _cachedChildNodes;
 
+  /// Returns a list of rects that bound the given selection.
+  ///
+  /// See [TextPainter.getBoxesForSelection] for more details.
+  List<Rect> getBoxesForSelection(TextSelection selection) =>
+    _textPainter.getBoxesForSelection(selection)
```

Comment:
nit: this method needs to call `layoutText` before accessing the text metrics to ensure the text metrics are populated in `_textPainter`.

Reply:
nit: per the style guide https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-braces-for-long-functions-and-methods
> Use a block (with braces) when a body would wrap onto more than one line (as opposed to using ⇒; the cases where you can use ⇒ are discussed in the previous two guidelines).